WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockUserTableViewCell.xib

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Suggestion] Same suggestion as [this one](https://github.com/wordpress-mobile/WordPress-iOS/pull/23178/files#r1602101190).